composer.json

```diff
@@ -1,46 +1,62 @@
 {
-    "name":         "behat/mink-selenium2-driver",
-    "description":  "Selenium2 (WebDriver) driver for Mink framework",
-    "keywords":     ["selenium", "webdriver", "javascript", "ajax", "testing", "browser"],
-    "homepage":     "http://mink.behat.org/",
-    "type":         "mink-driver",
-    "license":      "MIT",
-
+    "name": "behat/mink-selenium2-driver",
+    "description": "Selenium2 (WebDriver) driver for Mink framework",
+    "keywords": [
```

Comment:
If you want to do the reformatting of anything, then please create separate PR for that.